src/ui/src/app/dashboard/dashboard-body/main/leave-management/leave-history/leave-history.component.html

```diff
@@ -25,32 +36,48 @@
       <td>{{leaveRequest.fromDate}}</td>
       <td>{{leaveRequest.toDate}}</td>
       <td>{{leaveRequest.halfDay}}</td>
-      <td>{{leaveRequest.description}}</td>
+      <td><a href="javascript:void()" (click)="viewDescription(leaveRequest.description)">View </a></td>
```

Comment:
purpose?

Reply:
for the purpose of shortening the description to show description as a popup Modal when clicked